common/djangoapps/student/management/commands/transfer_students.py

```diff
@@ -52,7 +52,6 @@ class Command(TrackedCommand):
                     help="If True, try to transfer certificate items to the new course.")
     )
 
-    @transaction.commit_manually
```

Comment:
I do not know how this all plays out in the large scheme of how students and courses are handled in edx. I realize we would be changing behavior of this command, but I would rather see us keep the state of our data consistent. 

Let me pose this question what will be easier to track down in the future:
1) a command that fails to move all the students and provides a clear and concise error message
or
2) a command that succeeds, maybe puts something to stdout or stderr that someone will read or maybe not, then later when someone realizes that not all the students are moved they call in support or devops. Then someone has to comb through the data looking for said students, then they have to look to the code to try and figure out what happened. 

I just want to see us strive for both consistency in both our data and program behavior.

/soapbox